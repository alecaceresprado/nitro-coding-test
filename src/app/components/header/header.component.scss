@import "../../../styles/vars";

.header {
  background: $nitro-tile;
  top: 0;
  right: 0;
  left: 0;
  height: $header-height;

  &__logo{
    background: url(https://s3.amazonaws.com/cf-nitroengassets-prod/express/_global/logos/nitro-logo-orange-white.svg) 0 50% no-repeat;
    top: .5rem;
    left: .5rem;
    padding: 0;
    position: relative;
    height: 2rem;
    width: 4.75rem;
  }
}