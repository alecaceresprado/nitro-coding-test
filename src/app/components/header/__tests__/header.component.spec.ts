import { NO_ERRORS_SCHEMA } from '@angular/core';
import { TestBed, async, ComponentFixture } from '@angular/core/testing';

import { HeaderComponent } from '../header.component';

describe('HeaderComponent', () => {
  let fixture: ComponentFixture<HeaderComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      imports: [],
      declarations: [HeaderComponent],
      schemas: [NO_ERRORS_SCHEMA],
      providers: [],
    }).compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(HeaderComponent);
  });

  it('should create the app', () => {
    expect(fixture).toBeTruthy();
  });

  it('should render', () => {
    expect(fixture).toMatchSnapshot();
  });
});
