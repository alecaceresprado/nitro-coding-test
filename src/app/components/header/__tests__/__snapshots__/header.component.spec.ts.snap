// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderComponent should render 1`] = `
<app-header>
  <header
    class="header"
  >
    <div
      class="header__logo"
    />
  </header>
</app-header>
`;
